### DISCUSSION TITLE
Regarding project and assignments drop out.

### DISCUSSION THREAD

**Discussion by sorobedio** [seominjoon/kaist-ai605#155] [Q&A] [answered]
Professor said that for students who will do both all assignments and the project the two assignments with minimum scores will be dropped.
What if the sum of the scores of those two assignments is greater than what the student got in the project.
will the project score be dropped instead?

**Comment by seominjoon** [comment, marked as answer]
In this case, the project score will be dropped to maximize your final grade, so you don't have to worry it.
